code/modules/admin/admin.dm

```diff
@@ -10,6 +10,12 @@ var/global/floorIsLava = 0
 	for(var/client/C in GLOB.admins)
 		if(R_ADMIN & C.holder.rights)
 			to_chat(C, msg)
+/proc/message_mods(var/msg)
```

Comment:
Shouldn't this message people of rank mod -or higher- ? 
I can't see any reason you'd want to specifically message a moderator but not people with higher permissions. Admins, hosts, developers, etc.

As far as i can tell, the usage of this is going to limit the propagation of messages

Reply:
This should inherit. R_MOD is something that admins have.